velox/functions/lib/TimeUtils.h

```diff
@@ -104,4 +106,82 @@ struct InitSessionTimezone {
     timeZone_ = getTimeZoneFromConfig(config);
   }
 };
+
+/// Return day-of-year (DOY) of the first `firstDayOfWeek` in the year.
+/// If the `firstDayOfWeek` is Monday, it returns DOY of first Monday in
+/// the year. The returned DOY is a number from 1 to 7.
+///
+/// `firstDayOfWeek` is a 1-based weekday number starting with Sunday.
+///   (1 = Sunday, 2 = Monday, ..., 7 = Saturday).
+FOLLY_ALWAYS_INLINE
+uint32_t getDayOfFirstDayOfWeek(int32_t y, uint32_t firstDayOfWeek) {
```

Comment:
I feel the variable 'firstDayOfWeek' should be renamed as 'dayOfWeek' as it does not carry the **first** information. The function does the task to calculate the **first** one. How do you think?

Reply:
`dayOfWeek` usually represents the 7 days of the week such as Monday and Tuesday in common date library. `firstDayOfWeek` here represents which day is start of the week.

Reply:
I understand what you mean, you are right. `dayOfWeek` is appropriate here.